struqture-py/tests/spins/plus_minus_product.rs

```diff
@@ -617,3 +620,34 @@ fn test_from_error() {
         assert!(result.is_err());
     })
 }
+
+/// Test jordan_wigner() method of PlusMinusProduct
+#[test]
+fn test_jordan_wigner() {
+    pyo3::prepare_freethreaded_python();
+    pyo3::Python::with_gil(|py| {
+        let new_pmp = new_pp(py);
+        let pmp = new_pmp.call_method1("set_pauli", (0_u64, "+")).unwrap();
+        let fo = pmp.call_method0("jordan_wigner").unwrap();
+
+        let empty = bool::extract(fo.call_method0("is_empty").unwrap()).unwrap();
+        assert!(!empty);
+
+        let pp = pmp
+            .call_method0("to_pauli_product_list")
+            .unwrap()
+            .downcast::<PyList>()
+            .unwrap()
+            .get_item(0)
+            .unwrap()
+            .downcast::<PyTuple>()
+            .unwrap()
+            .get_item(0)
+            .unwrap();
+
+        let number_modes = usize::extract(fo.call_method0("number_modes").unwrap()).unwrap();
+        let number_spins =
+            usize::extract(pp.call_method0("current_number_spins").unwrap()).unwrap();
+        assert_eq!(number_modes, number_spins)
```

Comment:
Ah the pmp should also contain this method but I seem to have forgotten it, I'll pm you